app/models/poll.rb

```diff
@@ -1,5 +1,7 @@
 class Poll < ApplicationRecord
+  # belongs_to :lecture
```

Comment:
Why is this commented ? is it not longer true?

Reply:
- [ ] answer my question to review pass ;)

Reply:
this code isn't there anymore. I guess @MartinBuessemeyer added it there as an hint how to bind a poll to a lecture. But Polls are now implemented by another team and this code got removed.